game/ato/flightwaypoint.py

```diff
@@ -45,7 +45,7 @@ def __init__(
         # Only used in the waypoint list in the flight edit page. No sense
         # having three names. A short and long form is enough.
         self.description = ""
-        self.targets: Sequence[Union[MissionTarget, Unit]] = []
+        self.targets: list[GroundUnit] = []
```

Comment:
@DanAlbert i tried to figure out why `MissionTarget` was allowed and looked up all usages. It is never another object than GroundUnit assigned to this list so i just removed it. Tried a lot of different taskings and waypoint creations and all worked as i would assume. Maybe you can tell better why this was there in the first place :)